tests/protos/controllers/nested_mixed_template_proto/nested_mixed_template_proto.c

```diff
@@ -12,6 +12,15 @@
 
 #define TIME_STEP 32
 
+static void check_object_status(WbDeviceTag camera, int color, const char *message) {
```

Comment:
This `color` argument looks odd to me. It takes either 0 or 255 and doesn't correspond to a color but a grey level. I would replace it with a boolean called `visible` or similar.

Reply:
No, it is not possible to use a boolean value because the r/g/b value expected differs.
But I can change the argument name to `expected_grey_value` or similar.

Reply:
In a previous value the color where different. But now it seems possible to use a boolean value.